command_test.go

```diff
@@ -2058,3 +2058,40 @@ func TestFParseErrWhitelistSiblingCommand(t *testing.T) {
 	}
 	checkStringContains(t, output, "unknown flag: --unknown")
 }
+
+func TestPadding(t *testing.T) {
+	rootCmd := &Command{Use: "root", Run: emptyRun}
+	childCmd := &Command{Use: "child", Run: emptyRun}
+	longChildCmd := &Command{Use: "long-name-child-abcdefghijklmnopqrstuvwxyz", Run: emptyRun}
+	hiddenChildCmd := &Command{Use: "hidden-long-name-child-abcdefghijklmnopqrstuvwxyz", Hidden: true, Run: emptyRun}
+	deprecatedChildCmd := &Command{Use: "deprecated-long-name-child-abcdefghijklmnopqrstuvwxyz", Deprecated: "deprecated", Run: emptyRun}
```

Comment:
I believe that for the test to be useful, it is important that the deprecated command and hidden command have a longer `Use` field than the other ones.  This is the case now, but seems brittle.
Could you put a comment to say that it is needed?

Also, how about using `longChildCmd.Use + "-hidden"` for the `Use` of the hidden command, and similarly for the deprecated command?  That way we are sure they will be longer.